components/dialog/RotateEncryptionKeyDialog.vue

```diff
@@ -1,13 +1,15 @@
 <script>
+import { SNAPSHOT, NORMAN } from '@/config/types';
 import AsyncButton from '@/components/AsyncButton';
 import Card from '@/components/Card';
 import Banner from '@/components/Banner';
 import { exceptionToErrorsArray } from '@/utils/error';
-import { NORMAN } from '@/config/types';
+
 import { sortBy } from '@/utils/sort';
 import day from 'dayjs';
 import { escapeHtml } from '@/utils/string';
 import { DATE_FORMAT, TIME_FORMAT } from '@/store/prefs';
+import { set } from 'lodash';
```

Comment:
Just a note, 
- we're trying to move away from lodash, for this case we have a built in `set` somewhere in our codebase
- it's best to import the specific function via file `import set from 'lodash/set';`. in theory tree shacking should ensure anything not used in lodash isn't included in the final distribution, but that import pattern for single functions is used elsewhere